tests/python/relax/test_op_gradient_numeric.py

```diff
@@ -33,6 +33,7 @@ def relax_check_gradients(
     dev: tvm._ffi.runtime_ctypes.Device,
     output_shape: Union[Tuple, List[Tuple]],
     tuple_input: bool = False,
+    weight_dtype: str = "float32",
```

Comment:
Shall we rename it to output_dtype given that weights is a internal concept?

And I'm considering to add some documents for relax_check_gradients since now there are too many params

and adding more annotation in code